flyteadmin/auth/config/config.go

```diff
@@ -79,6 +79,10 @@ var (
 			},
 		},
 		AppAuth: OAuth2Options{
+			ExternalAuthServer: ExternalAuthorizationServer{
+				RetryAttempts:          5,
+				RetryDelayMilliseconds: config.Duration{Duration: 1000 * time.Millisecond},
```

Comment:
nit: since this type is already a duration no need to name it using "Milliseconds" since this could be any duration with any unit